data/json/itemgroups/tools.json

```
@@ -547,7 +548,8 @@
       [ "needle_curved", 30 ],
       [ "scissors", 1000 ],
       { "item": "sewing_kit", "prob": 10, "charges-min": 0 },
-      { "item": "tailors_kit", "prob": 120, "charges-min": 0 }
+      { "item": "tailors_kit", "prob": 120, "charges-min": 0 },
+      [ "kevlar_shears", 500 ]
```

```suggestion
      [ "kevlar_shears", 40 ]
```
Again, compare to what I'd call the flagship item of the set, a tailor's kit should be more common a spawn than kevlar shears in this group.